docs/reference-guides/best-practices/rancher-server/tips-for-running-rancher.md

```diff
@@ -14,8 +14,32 @@ If you are installing Rancher in a vSphere environment, refer to the best practi
 
 When you set up your high-availability Rancher installation, consider the following:
 
-### Run Rancher on a Separate Cluster
-Don't run other workloads or microservices in the Kubernetes cluster that Rancher is installed on.
+### Minimize Third-Party Software on the Upstream Cluster
+
+Running Rancher, especially as the number of managed clusters, nodes, and workloads increases, can place a significant load on core Kubernetes components like `etcd` and `kube-apiserver` on the upstream cluster. Third-party software can interfere with the performance of these components and Rancher itself, potentially causing issues.
+
+Every third-party application introduces a risk of interference. To minimize performance and incompatibility issues on the upstream cluster, avoid deploying any applications or components other than essential Kubernetes system components and Rancher.
+
+The following applications and components generally do not interfere with Rancher or Kubernetes system performance:
+ * Rancher internal components, such as Fleet
+ * Rancher extensions
+ * Cluster API components
+ * CNIs
+ * Cloud controller managers
+ * Observability and monitoring tools (with the exception of prometheus-rancher-exporter)
+ * the [SUSE Private Registry](https://documentation.suse.com/cloudnative/suse-private-registry/html/private-registry/index.html)
```

Comment:
I would rather see "Harbor" mentioned in this page as the 'general' case

Reply:
addressed in 0c35d2ca240